Unable to run map_generator on osm.pbf

**Describe the issue**
I am trying to run maps_generator to Saudi Arabia

I am running this command:

`python -m maps_generator --countries="Saudi Arabia_North, Saudi Arabia_South" --skip="Coastline"`

Getting this error:
```
python -m maps_generator --countries="Saudi Arabia_North, Saudi Arabia_South" --skip="Coastline"
Loading configuration from /mnt/c/users/k/Documents/organicmaps/tools/python/maps_generator/var/etc/map_generator.ini
[2023-09-02 15:40:01,923] INFO env Start setup ...
[2023-09-02 15:40:01,924] INFO env Check for the osmctools binaries...
[2023-09-02 15:40:02,055] ERROR env Uncaught exception: stat: path should be string, bytes, os.PathLike or integer, not NoneType
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/mnt/c/users/k/Documents/organicmaps/tools/python/maps_generator/__main__.py", line 250, in <module>
    main()
  File "/mnt/c/users/k/Documents/organicmaps/tools/python/maps_generator/__main__.py", line 232, in main
    env = Env(
  File "/mnt/c/users/k/Documents/organicmaps/tools/python/maps_generator/generator/env.py", line 390, in __init__
    for k, v in self.setup_osm_tools().items():
  File "/mnt/c/users/k/Documents/organicmaps/tools/python/maps_generator/generator/env.py", line 555, in setup_osm_tools
    if all([is_executable(t) for t in tmp_paths]):
  File "/mnt/c/users/k/Documents/organicmaps/tools/python/maps_generator/generator/env.py", line 555, in <listcomp>
    if all([is_executable(t) for t in tmp_paths]):
  File "/mnt/c/users/k/Documents/organicmaps/tools/python/maps_generator/utils/file.py", line 43, in is_executable
    return os.path.isfile(fpath) and os.access(fpath, os.X_OK)
  File "/usr/lib/python3.10/genericpath.py", line 30, in isfile
    st = os.stat(path)
TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType
```
osm.pbf file from:
`http://download.geofabrik.de/asia/gcc-states-latest.osm.pbf`

this is my map_generator.ini:

```
[Developer]
# Path to the `organicmaps` source code repository:
OMIM_PATH: /mnt/c/users/k/Documents/organicmaps
# A path with the generator_tool binary:
BUILD_PATH: ${Developer:OMIM_PATH}/../omim-build-release


[Main]
# A special small planet file will be downloaded if DEBUG is set to 1.
DEBUG: 0
# A main working directory. There is a subdirectory created for each generator run
# which contains the planet and other downloads, temporary build files, logs and completed MWMs.
MAIN_OUT_PATH: ${Developer:OMIM_PATH}/../maps_build
# Path for storing caches for nodes, ways, relations.
# If it's not set then caches are stored inside the directory of the current build.
# CACHE_PATH: ${Main:MAIN_OUT_PATH}/cache


[Generator tool]
# Path to the data/ folder in the repository:
USER_RESOURCE_PATH: ${Developer:OMIM_PATH}/data
# Features stage only parallelism level. Set to 0 for auto detection.
THREADS_COUNT_FEATURES_STAGE: 0


[Osm tools]
# Path to osmctools binaries:
OSM_TOOLS_PATH: ${Developer:OMIM_PATH}/../osmctools
# If the binaries are not found neither in the configured path nor system-wide,
# then the tools are built from the sources:
OSM_TOOLS_SRC_PATH: ${Developer:OMIM_PATH}/tools/osmctools


[Logging]
# maps_generator's general (python output only) log file path and name.
# More detailed logs that include output of the `generator_tool` binary
# are located in the `logs/` subdir of a particular build directory,
# e.g. `maps_build/2023_06_04__20_05_07/logs/`.
LOG_FILE_PATH: ${Main:MAIN_OUT_PATH}/generation.log


[External]
# Planet file location. It should be a dump of OSM data in osm.pbf format.
# By default its an entire planet from "planet.openstreetmap.org".
# Or set it to a particular country/region extract from e.g. [Geofabrik](http://download.geofabrik.de/index.html).
# Note that an entire planet generation takes 40+ hours on a 256GB RAM server (and 1TB+ disk space).
# Stick to smaller extracts unless you have a machine this large.
# Here and further, its possible to specify either an URL (to be downloaded automatically)
# or a local file path like file:///path/to/file.
# A sample URL to download a latest OSM dump for North Macedonia:
PLANET_URL: file://mnt/c/users/k/Downloads/gcc-states-latest.osm.pbf
# Location of the md5 checksum of the planet file:
PLANET_MD5_URL: file://mnt/c/users/k/Downloads/gcc-states-latest.osm.pbf.md5
# A base url to the latest_coasts.geom and latest_coasts.rawgeom files.
# For example, if PLANET_COASTS_URL = https://somesite.com/download/
# then the https://somesite.com/download/latest_coasts.geom url will be used to download latest_coasts.geom and
# the https://somesite.com/download/latest_coasts.rawgeom url will be used to download latest_coasts.rawgeom.
# Comment to skip getting the coastlines files.
# PLANET_COASTS_URL:
# Should be 'true' for an entire planet build to make a special routing section in World.mwm
# for alerting about absent regions without which the route can't be built.
NEED_BUILD_WORLD_ROADS: false
# Subway file location, see docs/SUBWAY_GENERATION.md if you want to generate your own file.
# Comment to disable subway layer generation.
#SUBWAY_URL: https://cdn.organicmaps.app/subway.json

# Location of the EXPERIMENTAL GTFS-extracted public transport transit files:
# TRANSIT_URL:

# Urls for production maps generation.
# UGC_URL:
# HOTELS_URL:
# PROMO_CATALOG_CITIES:
# POPULARITY_URL:
# FOOD_URL:
# FOOD_TRANSLATIONS_URL:
# SRTM_PATH:
# ISOLINES_PATH:
# UK_POSTCODES_URL:
# US_POSTCODES_URL:


[Stages]
# Set to 1 to update the entire OSM planet file (as taken from "planet.openstreetmap.org")
# via an osmupdate tool before the generation. Not for use with partial planet extracts.
NEED_PLANET_UPDATE: 0
# If you want to calculate diffs you need to specify where the old maps are.
DATA_ARCHIVE_DIR: ${Generator tool:USER_RESOURCE_PATH}
# How many versions in the archive to use for diff calculation:
DIFF_VERSION_DEPTH: 2


[Common]
# Default parallelism level for the most of jobs. Set to 0 for auto detection.
THREADS_COUNT: 0


[Stats]
# Path to rules for calculating statistics by type:
STATS_TYPES_CONFIG: ${Developer:OMIM_PATH}/tools/python/maps_generator/var/etc/stats_types_config.txt

```

And I've run these commands before:
```
./tools/unix/build_omim.sh -r desktop
./tools/unix/build_omim.sh -r generator_tool
./tools/unix/build_omim.sh -r world_roads_builder_tool
```


**System information:**
 - Operating system and its version: [Windows 11 + WSL=Ubuntu 22.04.]

`./tools/unix/build_omim.sh -r desktop` log:
```

-- Using compiler GNU 11.4.0
-- Using Unity Build with batch 50, export UNITY_DISABLE=1 or use -DUNITY_DISABLE=ON to disable it.
-- export COLORS_DISABLE=1 or use -DCOLORS_DISABLE=ON to disable colored compiler output.
Setting PLATFORM_LINUX to true
-- Build type: Release
-- Using ld.gold
-- Found the following ICU libraries:
--   uc (required)
--   i18n (required)
--   data (required)
-- ===========================================================================
--
-- Configuration
--   Generator .................. Ninja
--   Build type ................. Release
--   Prefix ..................... /usr/local
--   Shared libraries ........... OFF
--   Character type ............. char (UTF-8)
--   Library name postfix .......
--
--   Build documentation ........ OFF
--   Build examples ............. OFF
--   Build fuzzers .............. OFF
--   Build tests ................ OFF
--   Build tools (xmlwf) ........ OFF
--   Build pkg-config file ...... OFF
--   Install files .............. OFF
--
--   Features
--     // Advanced options, changes not advised
--     Attributes info .......... OFF
--     Context bytes ............ 1024
--     DTD support .............. ON
--     Large size ............... OFF
--     Minimum size ............. OFF
--     Namespace support ........ ON
--
--   Entropy sources
--     getrandom ................ 1
--     syscall SYS_getrandom .... 1
--     libbsd ................... OFF
--     /dev/random .............. ON
--
-- ===========================================================================
C compiler: GNU
-- Found python to use in qt/, shaders/ and 3party/: /usr/bin/python3.10





mkdir: cannot create directory ‘/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources’: Not a directory
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/countries-strings'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-default'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-mdpi_clear'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-hdpi_clear'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-xhdpi_clear'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-xxhdpi_clear'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-xxxhdpi_clear'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-6plus_clear'
cp: failed to access '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources/shaders_compiler': Not a directory
-- Configuring done
-- Generating done
-- Build files have been written to: /mnt/c/Users/k/Documents/omim-build-release

[35/103] Building CXX object geometry/CMakeFiles/geometry.dir/Unity/unity_0_cxx.cxx.o
In file included from /mnt/c/Users/k/Documents/organicmaps/geometry/robust_orientation.cpp:14,
                 from /mnt/c/Users/k/Documents/omim-build-release/geometry/CMakeFiles/geometry.dir/Unity/unity_0_cxx.cxx:37:
/mnt/c/Users/k/Documents/organicmaps/3party/robust/predicates.c: In function ‘double incircleadapt(double*, double*, double*, double*, double)’:
/mnt/c/Users/k/Documents/organicmaps/3party/robust/predicates.c:3103:44: warning: ‘cytablen’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 3103 |       temp16alen = scale_expansion_zeroelim(cytablen, cytab, cdytail, temp16a);
      |                    ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/c/Users/k/Documents/organicmaps/3party/robust/predicates.c:3061:44: warning: ‘cxtablen’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 3061 |       temp16alen = scale_expansion_zeroelim(cxtablen, cxtab, cdxtail, temp16a);
      |                    ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/c/Users/k/Documents/organicmaps/3party/robust/predicates.c:3006:44: warning: ‘bytcalen’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 3006 |       temp16alen = scale_expansion_zeroelim(bytcalen, bytca, bdytail, temp16a);
      |                    ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/c/Users/k/Documents/organicmaps/3party/robust/predicates.c:2909:44: warning: ‘aytbclen’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 2909 |       temp16alen = scale_expansion_zeroelim(aytbclen, aytbc, adytail, temp16a);
      |                    ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/c/Users/k/Documents/organicmaps/3party/robust/predicates.c:2867:44: warning: ‘axtbclen’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 2867 |       temp16alen = scale_expansion_zeroelim(axtbclen, axtbc, adxtail, temp16a);
      |                    ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/c/Users/k/Documents/organicmaps/3party/robust/predicates.c:2964:44: warning: ‘bxtcalen’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 2964 |       temp16alen = scale_expansion_zeroelim(bxtcalen, bxtca, bdxtail, temp16a);
      |                    ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[36/103] Building CXX object transit/CMakeFiles/transit.dir/Unity/unity_0_cxx.cxx.o
In file included from /mnt/c/Users/k/Documents/omim-build-release/transit/CMakeFiles/transit.dir/Unity/unity_0_cxx.cxx: :
/mnt/c/Users/k/Documents/organicmaps/transit/transit_graph_data.cpp: In member function ‘void routing::transit::DeserializerFromJson::operator()(routing::transit::FeatureIdentifiers&, const char*)’:
/mnt/c/Users/k/Documents/organicmaps/transit/transit_graph_data.cpp:189:41: warning: ‘osmIdNum’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  189 |   base::GeoObjectId const osmId(osmIdNum);
      |                                         ^
[53/103] Building CXX object indexer/CMakeFiles/indexer.dir/drules_struct.pb.cc.o
In file included from /usr/include/string.h:535,
                 from /mnt/c/Users/k/Documents/organicmaps/3party/protobuf/protobuf/src/google/protobuf/stubs/port.h:38,
                 from /mnt/c/Users/k/Documents/organicmaps/3party/protobuf/protobuf/src/google/protobuf/stubs/common.h:40,
                 from /mnt/c/Users/k/Documents/organicmaps/indexer/drules_struct.pb.h:9,
                 from /mnt/c/Users/k/Documents/organicmaps/indexer/drules_struct.pb.cc:5:
In function ‘void* memcpy(void*, const void*, size_t)’,
    inlined from ‘void google::protobuf::internal::ElementCopier<Element, true>::operator()(Element*, const Element*, int) [with Element = double]’ at /mnt/c/Users/k/Documents/organicmaps/3party/protobuf/protobuf/src/google/protobuf/repeated_field.h:1402:11,
    inlined from ‘void google::protobuf::RepeatedField<Element>::CopyArray(Element*, const Element*, int) [with Element = double]’ at /mnt/c/Users/k/Documents/organicmaps/3party/protobuf/protobuf/src/google/protobuf/repeated_field.h:1388:3 ,
    inlined from ‘void google::protobuf::RepeatedField<Element>::MoveArray(Element*, Element*, int) [with Element = double]’ at /mnt/c/Users/k/Documents/organicmaps/3party/protobuf/protobuf/src/google/protobuf/repeated_field.h:1382:12,
    inlined from ‘void google::protobuf::RepeatedField<Element>::Reserve(int) [with Element = double]’ at /mnt/c/Users/k/Documents/organicmaps/3party/protobuf/protobuf/src/google/protobuf/repeated_field.h:1363:14,
    inlined from ‘google::protobuf::RepeatedField<Element>::RepeatedField(const google::protobuf::RepeatedField<Element>&) [with Element = double]’ at /mnt/c/Users/k/Documents/organicmaps/3party/protobuf/protobuf/src/google/protobuf/repeated_field.h:1052:12,
    inlined from ‘DashDotProto::DashDotProto(const DashDotProto&)’ at /mnt/c/Users/k/Documents/organicmaps/indexer/drules_struct.pb.cc:221:7:
/usr/include/x86_64-linux-gnu/bits/string_fortified.h:29:33: warning: ‘void* __builtin_memcpy(void*, const void*, long unsigned int)’ offset [0, 7] is out of the bounds [0, 0] [-Warray-bounds]
   29 |   return __builtin___memcpy_chk (__dest, __src, __len,
      |          ~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
   30 |                                  __glibc_objsize0 (__dest));
      |                                  ~~~~~~~~~~~~~~~~~~~~~~~~~~
[54/103] Building CXX object storage/CMakeFiles/storage.dir/Unity/unity_0_cxx.cxx.o
In file included from /mnt/c/Users/k/Documents/omim-build-release/storage/CMakeFiles/storage.dir/Unity/unity_0_cxx.cxx:37:
/mnt/c/Users/k/Documents/organicmaps/storage/storage.cpp: In member function ‘void storage::Storage::RegisterCountryFiles(storage::LocalFilePtr)’:
/mnt/c/Users/k/Documents/organicmaps/storage/storage.cpp:931:60: warning: suggest braces around empty body in an ‘else’ statement [-Wempty-body]
  931 |       ASSERT_EQUAL(localFile.get(), existingFile.get(), ());
      |                                                            ^
[62/103] Building CXX object search/CMakeFiles/search.dir/Unity/unity_1_cxx.cxx.o
In file included from /mnt/c/Users/k/Documents/omim-build-release/search/CMakeFiles/search.dir/Unity/unity_1_cxx.cxx:9:
/mnt/c/Users/k/Documents/organicmaps/search/ranker.cpp: In member function ‘void search::Ranker::MakeRankerResults()’:
/mnt/c/Users/k/Documents/organicmaps/search/ranker.cpp:834:14: warning: unused variable ‘isViewportMode’ [-Wunused-variable]
  834 |   bool const isViewportMode = m_geocoderParams.m_mode == Mode::Viewport;
      |              ^~~~~~~~~~~~~~
[68/103] Building CXX object search/CMakeFiles/search.dir/Unity/unity_0_cxx.cxx.o
In file included from /mnt/c/Users/k/Documents/omim-build-release/search/CMakeFiles/search.dir/Unity/unity_0_cxx.cxx:10 :
/mnt/c/Users/k/Documents/organicmaps/search/processor.cpp: In member function ‘void search::Processor::SearchByFeatureId()’:
/mnt/c/Users/k/Documents/organicmaps/search/processor.cpp:491:76: warning: ‘version’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  491 |         if (guard.GetCountryFileName() == mwmName && guard.GetVersion() == version && fid < guard.GetNumFeatures())
      |                                                                            ^~~~~~~
[70/103] Building CXX object indexer/CMakeFiles/indexer.dir/Unity/unity_0_cxx.cxx.o
In file included from /mnt/c/Users/k/Documents/omim-build-release/indexer/CMakeFiles/indexer.dir/Unity/unity_0_cxx.cxx:47:
/mnt/c/Users/k/Documents/organicmaps/indexer/feature.cpp: In member function ‘void FeatureType::ParseHeader2()’:
/mnt/c/Users/k/Documents/organicmaps/indexer/feature.cpp:365:75: warning: suggest braces around empty body in an ‘else’ statement [-Wempty-body]
  365 |       ASSERT(headerGeomType == HeaderGeomType::Area || elemsCount > 1, ());
      |                                                                           ^
/mnt/c/Users/k/Documents/organicmaps/indexer/feature.cpp: In member function ‘std::string_view FeatureType::GetName(int8_t)’:
/mnt/c/Users/k/Documents/organicmaps/indexer/feature.cpp:836:86: warning: suggest braces around empty body in an ‘if’ statement [-Wempty-body]
  836 |     ASSERT(!name.empty() || m_id.m_mwmId.GetInfo()->GetType() == MwmInfo::COASTS, ());
      |                                                                                      ^
In file included from /mnt/c/Users/k/Documents/organicmaps/indexer/feature_to_osm.cpp:1,
                 from /mnt/c/Users/k/Documents/omim-build-release/indexer/CMakeFiles/indexer.dir/Unity/unity_0_cxx.cxx:63:
/mnt/c/Users/k/Documents/organicmaps/indexer/feature_to_osm.hpp: In member function ‘void MapUint32ToValue<Value>::ForEach(Fn&&) [with Fn = indexer::FeatureIdToGeoObjectIdSerDes::DeserializeV0ToMem<Reader>(Reader&, base::GeoObjectId::Type, indexer::FeatureIdToGeoObjectIdBimapMem&)::<lambda(uint32_t, uint64_t&)>; Value = long unsigned int]’:
/mnt/c/Users/k/Documents/organicmaps/indexer/feature_to_osm.hpp:335:29: warning: ‘value’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  335 |       memMap.Add(fid, base::GeoObjectId(type, serialId));
      |                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /mnt/c/Users/k/Documents/organicmaps/indexer/metadata_serdes.hpp:3,
                 from /mnt/c/Users/k/Documents/organicmaps/indexer/feature_meta.hpp:3,
                 from /mnt/c/Users/k/Documents/organicmaps/indexer/data_factory.hpp:3,
                 from /mnt/c/Users/k/Documents/organicmaps/indexer/mwm_set.hpp:2,
                 from /mnt/c/Users/k/Documents/organicmaps/indexer/altitude_loader.cpp:3,
                 from /mnt/c/Users/k/Documents/omim-build-release/indexer/CMakeFiles/indexer.dir/Unity/unity_0_cxx.cxx: :
/mnt/c/Users/k/Documents/organicmaps/coding/map_uint32_to_val.hpp:161:13: note: ‘value’ was declared here
  161 |       Value value;
      |             ^~~~~
[84/103] Building CXX object generator/CMakeFiles/generator.dir/final_processor_country.cpp.o
/mnt/c/Users/k/Documents/organicmaps/generator/final_processor_country.cpp: In member function ‘void generator::CountryFinalProcessor::ProcessBuildingParts()’:
/mnt/c/Users/k/Documents/organicmaps/generator/final_processor_country.cpp:199:21: warning: variable ‘buildingClassifType’ set but not used [-Wunused-but-set-variable]
  199 |   static auto const buildingClassifType = classificator.GetTypeByPath({"building"});
      |                     ^~~~~~~~~~~~~~~~~~~
/mnt/c/Users/k/Documents/organicmaps/generator/final_processor_country.cpp:200:21: warning: variable ‘buildingPartClassifType’ set but not used [-Wunused-but-set-variable]
  200 |   static auto const buildingPartClassifType = classificator.GetTypeByPath({"building:part"});
      |                     ^~~~~~~~~~~~~~~~~~~~~~~
/mnt/c/Users/k/Documents/organicmaps/generator/final_processor_country.cpp:201:21: warning: variable ‘buildingWithPartsClassifType’ set but not used [-Wunused-but-set-variable]
  201 |   static auto const buildingWithPartsClassifType = classificator.GetTypeByPath({"building", "has_parts"});
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/c/Users/k/Documents/organicmaps/generator/final_processor_country.cpp: In member function ‘void generator::CountryFinalProcessor::DropProhibitedSpeedCameras()’:
/mnt/c/Users/k/Documents/organicmaps/generator/final_processor_country.cpp:298:21: warning: variable ‘speedCameraType’ set but not used [-Wunused-but-set-variable]
  298 |   static auto const speedCameraType = classif().GetTypeByPath({"highway", "speed_camera"});
      |                     ^~~~~~~~~~~~~~~
[89/103] Building CXX object drape_frontend/CMakeFiles/drape_frontend.dir/Unity/unity_1_cxx.cxx.o
In file included from /mnt/c/Users/k/Documents/organicmaps/coding/geometry_coding.hpp:3,
                 from /mnt/c/Users/k/Documents/organicmaps/indexer/data_header.hpp:3,
                 from /mnt/c/Users/k/Documents/organicmaps/indexer/data_factory.hpp:2,
                 from /mnt/c/Users/k/Documents/organicmaps/indexer/mwm_set.hpp:2,
                 from /mnt/c/Users/k/Documents/organicmaps/indexer/feature_decl.hpp:3,
                 from /mnt/c/Users/k/Documents/organicmaps/drape_frontend/read_metaline_task.hpp:3,
                 from /mnt/c/Users/k/Documents/organicmaps/drape_frontend/metaline_manager.hpp:3,
                 from /mnt/c/Users/k/Documents/organicmaps/drape_frontend/metaline_manager.cpp:1,
                 from /mnt/c/Users/k/Documents/omim-build-release/drape_frontend/CMakeFiles/drape_frontend.dir/Unity/unity_1_cxx.cxx:5:
/mnt/c/Users/k/Documents/organicmaps/geometry/point2d.hpp: In function ‘void df::CacheUserLines(ref_ptr<dp::GraphicsContext>, const df::TileKey&, ref_ptr<dp::TextureManager>, const TrackIdCollection&, const UserLinesRenderCollection&, dp::Batcher&)’:
/mnt/c/Users/k/Documents/organicmaps/geometry/point2d.hpp:39:86: warning: ‘lastAddedPoint.m2::Point<double>::y’ may be used uninitialized in this function [-Wmaybe-uninitialized]
   39 |   T SquaredLength(Point<T> const & p) const { return base::Pow2(x - p.x) + base::Pow2(y - p.y); }
      |                                                                            ~~~~~~~~~~^~~~~~~~~
In file included from /mnt/c/Users/k/Documents/omim-build-release/drape_frontend/CMakeFiles/drape_frontend.dir/Unity/unity_1_cxx.cxx:87:
/mnt/c/Users/k/Documents/organicmaps/drape_frontend/user_mark_shapes.cpp:333:14: note: ‘lastAddedPoint.m2::Point<double>::y’ was declared here
  333 |   m2::PointD lastAddedPoint;
      |              ^~~~~~~~~~~~~~
In file included from /mnt/c/Users/k/Documents/organicmaps/coding/geometry_coding.hpp:3,
                 from /mnt/c/Users/k/Documents/organicmaps/indexer/data_header.hpp:3,
                 from /mnt/c/Users/k/Documents/organicmaps/indexer/data_factory.hpp:2,
                 from /mnt/c/Users/k/Documents/organicmaps/indexer/mwm_set.hpp:2,
                 from /mnt/c/Users/k/Documents/organicmaps/indexer/feature_decl.hpp:3,
                 from /mnt/c/Users/k/Documents/organicmaps/drape_frontend/read_metaline_task.hpp:3,
                 from /mnt/c/Users/k/Documents/organicmaps/drape_frontend/metaline_manager.hpp:3,
                 from /mnt/c/Users/k/Documents/organicmaps/drape_frontend/metaline_manager.cpp:1,
                 from /mnt/c/Users/k/Documents/omim-build-release/drape_frontend/CMakeFiles/drape_frontend.dir/Unity/unity_1_cxx.cxx:5:
/mnt/c/Users/k/Documents/organicmaps/geometry/point2d.hpp:39:64: warning: ‘lastAddedPoint.m2::Point<double>::x’ may be used uninitialized in this function [-Wmaybe-uninitialized]
   39 |   T SquaredLength(Point<T> const & p) const { return base::Pow2(x - p.x) + base::Pow2(y - p.y); }
      |                                                      ~~~~~~~~~~^~~~~~~~~
In file included from /mnt/c/Users/k/Documents/omim-build-release/drape_frontend/CMakeFiles/drape_frontend.dir/Unity/unity_1_cxx.cxx:87:
/mnt/c/Users/k/Documents/organicmaps/drape_frontend/user_mark_shapes.cpp:333:14: note: ‘lastAddedPoint.m2::Point<double>::x’ was declared here
  333 |   m2::PointD lastAddedPoint;
      |              ^~~~~~~~~~~~~~
[90/103] Building CXX object drape_frontend/CMakeFiles/drape_frontend.dir/Unity/unity_0_cxx.cxx.o
In file included from /mnt/c/Users/k/Documents/omim-build-release/drape_frontend/CMakeFiles/drape_frontend.dir/Unity/unity_0_cxx.cxx:37:
/mnt/c/Users/k/Documents/organicmaps/drape_frontend/arrow3d.cpp: In member function ‘std::pair<glm::mat<4, 4, float, glm::packed_highp>, glm::mat<4, 4, float, glm::packed_highp> > df::Arrow3d::CalculateTransform(const ScreenBase&, float, float, dp::ApiVersion) const’:
/mnt/c/Users/k/Documents/organicmaps/drape_frontend/arrow3d.cpp:507:11: warning: ‘void* memcpy(void*, const void*, size_t)’ copying an object of non-trivial type ‘glm::mat4’ {aka ‘struct glm::mat<4, 4, float, glm::packed_highp>’} from an array of ‘const struct math::Matrix<float, 4, 4>’ [-Wclass-memaccess]
  507 |     memcpy(&pTo3dView, &m, sizeof(pTo3dView));
      |     ~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /mnt/c/Users/k/Documents/organicmaps/3party/glm/glm/gtc/../detail/../mat4x4.hpp:8,
                 from /mnt/c/Users/k/Documents/organicmaps/3party/glm/glm/gtc/../detail/func_matrix.hpp:32,
                 from /mnt/c/Users/k/Documents/organicmaps/3party/glm/glm/gtc/../detail/type_mat3x3.inl:4,
                 from /mnt/c/Users/k/Documents/organicmaps/3party/glm/glm/gtc/../detail/type_mat3x3.hpp:189,
                 from /mnt/c/Users/k/Documents/organicmaps/3party/glm/glm/gtc/../mat3x3.hpp:8,
                 from /mnt/c/Users/k/Documents/organicmaps/3party/glm/glm/gtc/quaternion.hpp:17,
                 from /mnt/c/Users/k/Documents/organicmaps/drape/glsl_types.hpp:7,
                 from /mnt/c/Users/k/Documents/organicmaps/drape_frontend/area_shape.hpp:6,
                 from /mnt/c/Users/k/Documents/organicmaps/drape_frontend/apply_feature_functors.cpp:3,
                 from /mnt/c/Users/k/Documents/omim-build-release/drape_frontend/CMakeFiles/drape_frontend.dir/Unity/unity_0_cxx.cxx:33:
/mnt/c/Users/k/Documents/organicmaps/3party/glm/glm/gtc/../detail/../detail/type_mat4x4.hpp:15:16: note: ‘glm::mat4’ {aka ‘struct glm::mat<4, 4, float, glm::packed_highp>’} declared here
   15 |         struct mat<4, 4, T, P>
      |                ^~~~~~~~~~~~~~~
In file included from /mnt/c/Users/k/Documents/omim-build-release/drape_frontend/CMakeFiles/drape_frontend.dir/Unity/unity_0_cxx.cxx:43:
/mnt/c/Users/k/Documents/organicmaps/drape_frontend/circles_pack_shape.cpp: In member function ‘void df::CirclesPackHandle::Clear()’:
/mnt/c/Users/k/Documents/organicmaps/drape_frontend/circles_pack_shape.cpp:133:9: warning: ‘void* memset(void*, int, size_t)’ clearing an object of non-trivial type ‘struct df::CirclesPackDynamicVertex’; use assignment or value-initialization instead [-Wclass-memaccess]
  133 |   memset(m_buffer.data(), 0, m_buffer.size() * sizeof(CirclesPackDynamicVertex));
      |   ~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /mnt/c/Users/k/Documents/organicmaps/drape_frontend/backend_renderer.cpp:5,
                 from /mnt/c/Users/k/Documents/omim-build-release/drape_frontend/CMakeFiles/drape_frontend.dir/Unity/unity_0_cxx.cxx:39:
/mnt/c/Users/k/Documents/organicmaps/drape_frontend/circles_pack_shape.hpp:34:8: note: ‘struct df::CirclesPackDynamicVertex’ declared here
   34 | struct CirclesPackDynamicVertex
      |        ^~~~~~~~~~~~~~~~~~~~~~~~
In file included from /mnt/c/Users/k/Documents/omim-build-release/drape_frontend/CMakeFiles/drape_frontend.dir/Unity/unity_0_cxx.cxx:89:
/mnt/c/Users/k/Documents/organicmaps/drape_frontend/gui/skin.cpp: In function ‘dp::Anchor gui::{anonymous}::ParseValueAnchor(std::string_view)’:
/mnt/c/Users/k/Documents/organicmaps/drape_frontend/gui/skin.cpp:40:22: warning: suggest braces around empty body in an ‘else’ statement [-Wempty-body]
   40 |     ASSERT(false, ());
      |                      ^
/mnt/c/Users/k/Documents/organicmaps/drape_frontend/gui/skin.cpp: In member function ‘void gui::{anonymous}::ResolverParser::Parse(std::string_view, const char*)’:
/mnt/c/Users/k/Documents/organicmaps/drape_frontend/gui/skin.cpp:97:26: warning: suggest braces around empty body in an ‘else’ statement [-Wempty-body]
   97 |         ASSERT(false, ());
      |                          ^
/mnt/c/Users/k/Documents/organicmaps/drape_frontend/gui/skin.cpp: In member function ‘bool gui::{anonymous}::SkinLoader::Push(std::string_view)’:
/mnt/c/Users/k/Documents/organicmaps/drape_frontend/gui/skin.cpp:143:26: warning: suggest braces around empty body in an ‘else’ statement [-Wempty-body]
  143 |         ASSERT(false, ());
      |                          ^
/mnt/c/Users/k/Documents/organicmaps/drape_frontend/gui/skin.cpp:150:26: warning: suggest braces around empty body in an ‘else’ statement [-Wempty-body]
  150 |         ASSERT(false, ());
      |                          ^
/mnt/c/Users/k/Documents/organicmaps/drape_frontend/gui/skin.cpp:161:26: warning: suggest braces around empty body in an ‘else’ statement [-Wempty-body]
  161 |         ASSERT(false, ());
      |                          ^
In file included from /mnt/c/Users/k/Documents/omim-build-release/drape_frontend/CMakeFiles/drape_frontend.dir/Unity/unity_0_cxx.cxx:95:
/mnt/c/Users/k/Documents/organicmaps/drape_frontend/line_shape_helper.cpp: In function ‘void df::ConstructLineSegments(const std::vector<m2::Point<double> >&, const std::vector<glm::vec<4, float, glm::packed_highp> >&, std::vector<df::LineSegment>&)’:
/mnt/c/Users/k/Documents/organicmaps/drape_frontend/line_shape_helper.cpp:93:61: warning: suggest braces around empty body in an ‘if’ statement [-Wempty-body]
   93 |     ASSERT_EQUAL(segmentsColors.size() + 1, path.size(), ());
      |                                                             ^
[92/103] Building CXX object map/CMakeFiles/map.dir/Unity/unity_0_cxx.cxx.o
In file included from /mnt/c/Users/k/Documents/omim-build-release/map/CMakeFiles/map.dir/Unity/unity_0_cxx.cxx:11:
/mnt/c/Users/k/Documents/organicmaps/map/bookmark_manager.cpp: In lambda function:
/mnt/c/Users/k/Documents/organicmaps/map/bookmark_manager.cpp:1879:64: warning: suggest braces around empty body in an  else’ statement [-Wempty-body]
 1879 |         ASSERT(false, ("Unsupported bookmarks extension", ext));
      |                                                                ^
In file included from /mnt/c/Users/k/Documents/organicmaps/3party/agg/agg_conv_curve.h:24,
                 from /mnt/c/Users/k/Documents/organicmaps/map/chart_generator.cpp:9,
                 from /mnt/c/Users/k/Documents/omim-build-release/map/CMakeFiles/map.dir/Unity/unity_0_cxx.cxx:13:
/mnt/c/Users/k/Documents/organicmaps/3party/agg/agg_curves.h: In member function ‘unsigned int agg::conv_curve<VertexSource, Curve3, Curve4>::vertex(double*, double*) [with VertexSource = agg::path_base<agg::vertex_block_storage<double> >; Curve3 = agg::curve3; Curve4 = agg::curve4]’:
/mnt/c/Users/k/Documents/organicmaps/3party/agg/agg_curves.h:611:33: warning: ‘ct2_y’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  611 |                 m_curve_inc.init(x1, y1, x2, y2, x3, y3, x4, y4);
      |                 ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /mnt/c/Users/k/Documents/organicmaps/map/chart_generator.cpp:9,
                 from /mnt/c/Users/k/Documents/omim-build-release/map/CMakeFiles/map.dir/Unity/unity_0_cxx.cxx:13:
/mnt/c/Users/k/Documents/organicmaps/3party/agg/agg_conv_curve.h:158:16: note: ‘ct2_y’ was declared here
  158 |         double ct2_y;
      |                ^~~~~
In file included from /mnt/c/Users/k/Documents/organicmaps/3party/agg/agg_conv_curve.h:24,
                 from /mnt/c/Users/k/Documents/organicmaps/map/chart_generator.cpp:9,
                 from /mnt/c/Users/k/Documents/omim-build-release/map/CMakeFiles/map.dir/Unity/unity_0_cxx.cxx:13:
/mnt/c/Users/k/Documents/organicmaps/3party/agg/agg_curves.h:611:33: warning: ‘ct2_x’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  611 |                 m_curve_inc.init(x1, y1, x2, y2, x3, y3, x4, y4);
      |                 ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /mnt/c/Users/k/Documents/organicmaps/map/chart_generator.cpp:9,
                 from /mnt/c/Users/k/Documents/omim-build-release/map/CMakeFiles/map.dir/Unity/unity_0_cxx.cxx:13:
/mnt/c/Users/k/Documents/organicmaps/3party/agg/agg_conv_curve.h:157:16: note: ‘ct2_x’ was declared here
  157 |         double ct2_x;
      |                ^~~~~
In file included from /mnt/c/Users/k/Documents/organicmaps/3party/agg/agg_conv_curve.h:24,
                 from /mnt/c/Users/k/Documents/organicmaps/map/chart_generator.cpp:9,
                 from /mnt/c/Users/k/Documents/omim-build-release/map/CMakeFiles/map.dir/Unity/unity_0_cxx.cxx:13:
/mnt/c/Users/k/Documents/organicmaps/3party/agg/agg_curves.h:611:33: warning: ‘end_y’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  611 |                 m_curve_inc.init(x1, y1, x2, y2, x3, y3, x4, y4);
      |                 ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /mnt/c/Users/k/Documents/organicmaps/map/chart_generator.cpp:9,
                 from /mnt/c/Users/k/Documents/omim-build-release/map/CMakeFiles/map.dir/Unity/unity_0_cxx.cxx:13:
/mnt/c/Users/k/Documents/organicmaps/3party/agg/agg_conv_curve.h:160:16: note: ‘end_y’ was declared here
  160 |         double end_y;
      |                ^~~~~
In file included from /mnt/c/Users/k/Documents/organicmaps/3party/agg/agg_conv_curve.h:24,
                 from /mnt/c/Users/k/Documents/organicmaps/map/chart_generator.cpp:9,
                 from /mnt/c/Users/k/Documents/omim-build-release/map/CMakeFiles/map.dir/Unity/unity_0_cxx.cxx:13:
/mnt/c/Users/k/Documents/organicmaps/3party/agg/agg_curves.h:611:33: warning: ‘end_x’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  611 |                 m_curve_inc.init(x1, y1, x2, y2, x3, y3, x4, y4);
      |                 ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /mnt/c/Users/k/Documents/organicmaps/map/chart_generator.cpp:9,
                 from /mnt/c/Users/k/Documents/omim-build-release/map/CMakeFiles/map.dir/Unity/unity_0_cxx.cxx:13:
/mnt/c/Users/k/Documents/organicmaps/3party/agg/agg_conv_curve.h:159:16: note: ‘end_x’ was declared here
  159 |         double end_x;
      |                ^~~~~
In file included from /mnt/c/Users/k/Documents/omim-build-release/map/CMakeFiles/map.dir/Unity/unity_0_cxx.cxx:15:
/mnt/c/Users/k/Documents/organicmaps/map/elevation_info.cpp: In constructor ‘ElevationInfo::ElevationInfo(const Track&)’:
/mnt/c/Users/k/Documents/organicmaps/map/elevation_info.cpp:60:3: warning: ‘difficulty’ may be used uninitialized in this function [-Wmaybe-uninitialized]
   60 |   if (difficulty > kMaxDifficulty)
      |   ^~
[103/103] Linking CXX executable OMaps
```
`./tools/unix/build_omim.sh -r generator_tool` log:

```
-- Using compiler GNU 11.4.0
-- Using Unity Build with batch 50, export UNITY_DISABLE=1 or use -DUNITY_DISABLE=ON to disable it.
-- export COLORS_DISABLE=1 or use -DCOLORS_DISABLE=ON to disable colored compiler output.
Setting PLATFORM_LINUX to true
-- Build type: Release
-- Using ld.gold
-- Found the following ICU libraries:
--   uc (required)
--   i18n (required)
--   data (required)
-- ===========================================================================
--
-- Configuration
--   Generator .................. Ninja
--   Build type ................. Release
--   Prefix ..................... /usr/local
--   Shared libraries ........... OFF
--   Character type ............. char (UTF-8)
--   Library name postfix .......
--
--   Build documentation ........ OFF
--   Build examples ............. OFF
--   Build fuzzers .............. OFF
--   Build tests ................ OFF
--   Build tools (xmlwf) ........ OFF
--   Build pkg-config file ...... OFF
--   Install files .............. OFF
--
--   Features
--     // Advanced options, changes not advised
--     Attributes info .......... OFF
--     Context bytes ............ 1024
--     DTD support .............. ON
--     Large size ............... OFF
--     Minimum size ............. OFF
--     Namespace support ........ ON
--
--   Entropy sources
--     getrandom ................ 1
--     syscall SYS_getrandom .... 1
--     libbsd ................... OFF
--     /dev/random .............. ON
--
-- ===========================================================================
C compiler: GNU
-- Found python to use in qt/, shaders/ and 3party/: /usr/bin/python3.10
mkdir: cannot create directory ‘/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources’: Not a directory
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/countries-strings'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-default'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-mdpi_clear'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-hdpi_clear'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-xhdpi_clear'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-xxhdpi_clear'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-xxxhdpi_clear'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-6plus_clear'
cp: failed to access '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources/shaders_compiler': Not a directory
-- Configuring done
-- Generating done
-- Build files have been written to: /mnt/c/Users/k/Documents/omim-build-release

[0/2] Building CXX object generator/generator_tool/CMakeFiles/generator_tool.dir/Unity/unity_0_cxx.cxx.o
[2/2] Linking CXX executable generator_tool
```
`./tools/unix/build_omim.sh -r world_roads_builder_tool` log:
```
-- Using compiler GNU 11.4.0
-- Using Unity Build with batch 50, export UNITY_DISABLE=1 or use -DUNITY_DISABLE=ON to disable it.
-- export COLORS_DISABLE=1 or use -DCOLORS_DISABLE=ON to disable colored compiler output.
Setting PLATFORM_LINUX to true
-- Build type: Release
-- Using ld.gold
-- Found the following ICU libraries:
--   uc (required)
--   i18n (required)
--   data (required)
-- ===========================================================================
--
-- Configuration
--   Generator .................. Ninja
--   Build type ................. Release
--   Prefix ..................... /usr/local
--   Shared libraries ........... OFF
--   Character type ............. char (UTF-8)
--   Library name postfix .......
--
--   Build documentation ........ OFF
--   Build examples ............. OFF
--   Build fuzzers .............. OFF
--   Build tests ................ OFF
--   Build tools (xmlwf) ........ OFF
--   Build pkg-config file ...... OFF
--   Install files .............. OFF
--
--   Features
--     // Advanced options, changes not advised
--     Attributes info .......... OFF
--     Context bytes ............ 1024
--     DTD support .............. ON
--     Large size ............... OFF
--     Minimum size ............. OFF
--     Namespace support ........ ON
--
--   Entropy sources
--     getrandom ................ 1
--     syscall SYS_getrandom .... 1
--     libbsd ................... OFF
--     /dev/random .............. ON
--
-- ===========================================================================
C compiler: GNU
-- Found python to use in qt/, shaders/ and 3party/: /usr/bin/python3.10
mkdir: cannot create directory ‘/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources’: Not a directory
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/countries-strings'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-default'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-mdpi_clear'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-hdpi_clear'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-xhdpi_clear'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-xxhdpi_clear'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-xxxhdpi_clear'
cp: cannot overwrite non-directory '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources' with directory '/mnt/c/Users/k/Documents/organicmaps/data/resources-6plus_clear'
cp: failed to access '/mnt/c/Users/k/Documents/omim-build-release/OMaps.app/Contents/Resources/shaders_compiler': Not a directory
-- Configuring done
-- Generating done
-- Build files have been written to: /mnt/c/Users/k/Documents/omim-build-release

[4/4] Linking CXX executable world_roads_builder_tool
```

--- a/tools/python/maps_generator/utils/file.py
+++ b/tools/python/maps_generator/utils/file.py
@@ -40,7 +40,7 @@ def file_uri_to_path(url : AnyStr) -> AnyStr:
     return file_path
 
 def is_executable(fpath: AnyStr) -> bool:
-    return os.path.isfile(fpath) and os.access(fpath, os.X_OK)
+    return fpath is not None and os.path.isfile(fpath) and os.access(fpath, os.X_OK)
 
 
 @functools.lru_cache()

